docs/build/build-macOS.md

```diff
@@ -0,0 +1,30 @@
+# Building SRT for macOS
+
+[Homebrew](https://brew.sh/) supports "srt" formula.
+
+```shell
+brew update
+brew install srt
+```
+
+If you prefer using a head commit of `master` branch, you should add `--HEAD` option
+to `brew` command.
+
+```shell
+brew install --HEAD srt
+```
+
+Also, SRT can be built with `cmake` and `make` on Mac.
+Install cmake and openssl with development files from "brew". Note that the
+system version of OpenSSL is inappropriate, although you should be able to
+use any newer version compiled from sources, if you prefer.
```

Comment:
```suggestion
SRT can also be built with `cmake` and `make` on Mac.
Install `cmake` and `openssl` with development files using the `brew` command. Note that the
system version of OpenSSL is inappropriate [**??? what does "inappropriate" mean?**], although you should be able to
use any newer version compiled from sources if you prefer.
```

Reply:
@mbakholdina Please see question above.

Reply:
Likely whether particular version of OpenSSL is not working with SRT, may depend on the version. So I'd recommend leaving a recommendation that OpenSSL better be installed as latest version from the "brew" system rather than relying on the version that is presently installed in the system.